setup.py

```diff
@@ -19,7 +19,7 @@ def read(fname):
 # This call to setup() does all the work
 setup(
     name="ansible-deployer",
-    version="0.0.24",
+    version="0.0.27",
```

Comment:
it's always fine, but it's also one of the things that may result in conflicts so.. sometimes is better not to touch it :)